README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
+| Cancelling of builds  |       |      |    |
+| Chaining of builds    |       |      |    |
 | Image Caching         |       |      |    |
```

Comment:
@adambkaplan as discussed previously IMO DEVEX should either facilitate this with the existing stories or have subsequent ones that make sure our solution is consumable via the `buildah` binary so buildv2 can leverage